ert/data/record/_transformation.py

```diff
@@ -46,6 +62,14 @@ async def transform_input(
     async def transform_output(self, mime: str, location: Path) -> Record:
         return await _load_record_from_file(location, mime)
 
+    async def transform_output_sequence(
+        self, mime: str, location: Path
```

Comment:
Regarding the discussion earlier, I think this is a reasonable compromise for now :+1: